### DISCUSSION TITLE
gcc & gcc@5 links conflict

### DISCUSSION THREAD

**Discussion by jacktose** [Homebrew/discussions#809] [Linux] [answered]
I tried to reinstall gcc to update to the new dependency setup from Homebrew/brew#10512 and Homebrew/brew#10569—the `gcc` formula now depends on `gcc@5`. But it seems that `gcc` and `gcc@5` want to overwrite each other's symlinks. [Here's the gist of it.][gist]

I'm guessing the solution is to `brew link --overwrite` one or the other of them, but which one is supposed to win?

Is this be something weird about my machine, or is everyone now seeing this on linuxbrew? If the latter, can it be better handled at install time?

[#10512]: /Homebrew/brew/pull/10512
[#10569]: /Homebrew/brew/pull/10569
[gist]: https://gist.github.com/jacktose/0f7251c261cb3ec63a3d271d186ed32d

**Comment by iMichka** [comment, marked as answer]
You can use the overwrite method for the time being. Please follow https://github.com/Homebrew/linuxbrew-core/pull/21380, I am going to merge that PR next week. This should do a big cleanup and solve your issue too.

**Reply by jacktose** [reply]
Following, thanks.
Does it matter whether I overwrite one or the other? Or unlink both, since it's only there as a build dep?

**Reply by iMichka** [reply]
Both gcc and gcc@5 are the same version right now (this is going to change soon). I guess the best is to use `gcc@5`.
